tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -99,125 +103,92 @@ Verify Alert RHODS-PVC-Usage-At-100 Is Fired When User PVC Is At 100 Percent
 
     [Teardown]    Teardown PVC Alert Test
 
-Verify Alert "RHODS Route Error Burn Rate" Is Fired When Traefik Is Down    # robocop: disable:too-long-test-case
-    [Documentation]    Verifies that alert "RHODS Route Error Burn Rate" is fired when traefik-proxy is not working
+Verify Alerts Are Fired When Traefik Is Down    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that alerts "RHODS JupyterHub Probe Success Burn Rate" and "RHODS Probe
+    Success Burn Rate" alerts are fired when traefik-proxy is not working
     [Tags]    Tier3
     ...       ODS-738
 
     Skip    msg=This alert was disabled in RHODS 1.3.0. More info at RHODS-2101
 
     Skip Test If Alert Is Already Firing    ${RHODS_PROMETHEUS_URL}
     ...    ${RHODS_PROMETHEUS_TOKEN}
-    ...    SLOs-haproxy_backend_http_responses_total
-    ...    RHODS Route Error Burn Rate
+    ...    SLOs-probe_success
+    ...    RHODS JupyterHub Probe Success Burn Rate
     ...    alert-duration=120
 
     ODS.Scale Deployment    redhat-ods-operator    rhods-operator    replicas=0
     ODS.Scale Deployment    redhat-ods-applications    traefik-proxy    replicas=0
 
     Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
     ...    ${RHODS_PROMETHEUS_TOKEN}
-    ...    SLOs-haproxy_backend_http_responses_total
-    ...    RHODS Route Error Burn Rate
+    ...    SLOs-probe_success
+    ...    RHODS JupyterHub Probe Success Burn Rate
+    ...    alert-duration=120
+    ...    timeout=40 min
+
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    SLOs-probe_success
+    ...    RHODS Probe Success Burn Rate
     ...    alert-duration=120
     ...    timeout=40 min
 
     ODS.Restore Default Deployment Sizes
 
     Prometheus.Wait Until Alert Is Not Firing    ${RHODS_PROMETHEUS_URL}
     ...    ${RHODS_PROMETHEUS_TOKEN}
-    ...    SLOs-haproxy_backend_http_responses_total
-    ...    RHODS Route Error Burn Rate
+    ...    SLOs-probe_success
+    ...    RHODS JupyterHub Probe Success Burn Rate
+    ...    alert-duration=120
+    ...    timeout=5 min
+
+    Prometheus.Wait Until Alert Is Not Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    SLOs-probe_success
+    ...    RHODS Probe Success Burn Rate
     ...    alert-duration=120
     ...    timeout=5 min
 
     [Teardown]    ODS.Restore Default Deployment Sizes
 
-Verify Alert "RHODS Route Error Burn Rate" Is Fired When RHODS Dashboard Is Down    # robocop: disable:too-long-test-case
-    [Documentation]    Verifies that alert "RHODS Route Error Burn Rate" is fired when rhods-dashboard is not working
+Verify Alerts are Fired When RHODS Dashboard Is Down    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that alert "RHODS Dashboard Route Error Burn Rate" and "RHODS Probe Success Burn Rate"
+    are fired when rhods-dashboard is not working
```

Comment:
It should be (3 dots followed by 4 spaces) `... are fired when rhods-dashboard is not working`

Reply:
Updated